content/influxdb/v2.0/reference/flux/stdlib/contrib/tickscript/alert.md

```diff
@@ -0,0 +1,143 @@
+---
+title: tickscript.alert() function
+description: >
+  The `tickscript.alert()` function function identifies events of varying severity levels
+  and writes them to the `statuses` measurement in the InfluxDB
+  `_monitoring` system bucket.
+menu:
+  influxdb_2_0_ref:
+    name: tickscript.alert
+    parent: TICKscript
+weight: 302
+related:
+  - /{{< latest "kapacitor" >}}/nodes/alert_node/, Kapacitor AlertNode
+---
+
+The `tickscript.alert()` function identifies events of varying severity levels
+and writes them to the `statuses` measurement in the InfluxDB
+[`_monitoring` system bucket](/influxdb/v2.0/reference/internals/system-buckets/).
+
+_This function is comparable to the [Kapacitor AlertNode](/{{< latest "kapacitor" >}}/nodes/alert_node/)._
+
+```js
+import "contrib/bonitoo-io/tickscript"
+
+tickscript.alert(
+    check,
+    id: (r) => "${r._check_id}",
+    details: (r) => "",
+    message: (r) => "Threshold Check: ${r._check_name} is: ${r._level}",
+    crit: (r) => false,
+    warn: (r) => false,
+    info: (r) => false,
+    ok: (r) => true,
+    topic: ""
+)
+```
+
+## Parameters
+
+### check
+({{< req >}})
+InfluxDB check data.
+_See [`tickscript.defineCheck()`](/influxdb/v2.0/reference/flux/stdlib/contrib/tickscript/definecheck/)._
+
+_**Data type:** Record_
+
+### id
+Function to return the InfluxDB check ID provided by the [`check` record](#check).
```

Comment:
A thought: `Returns...` might be more readable than `Function to return`.

Reply:
Fixed